monero-outreach-docs/translations/pt-br/storyboard/fundamental_pt_br.md

```diff
@@ -0,0 +1,79 @@
+
+## Privacidade para SUAS Quantias, SUA Moeda e SEUS Valores
+
+Seu dinheiro só pode ser visto por você e por aqueles que você permitir. Ao longo do tempo, com o sistema financeiro tornando-se cada vez mais monopolizado e digitalizado, passamos a aceitar que bancos e governos vejam todas as vezes que passamos o cartão de crédito, o saldo em nossas contas e os saques que realizamos. Isso é perigoso. Todos têm aceitado muito livremente essa invasão em sua vida cotidiana, quando a resistência teria sido a resposta mais lógica.
+
+Temos testemunhado, mais e mais, esforços dos governos para colocar o mundo todo sob vigilância, com a [decisão sobre o regime de espionagem do Reino Unido](https://www.eff.org/deeplinks/2018/09/uk-surveillance-regime-violated-human-rights) sendo o exemplo mais recente. Mesmo que essas operações secretas e chocantes sejam tornadas públicas, os perpetradores continuam a se envolver nessas violações, muitas vezes mudando seus métodos.
+
+## Fungibilidade como uma forma de resistência à vigilância
+
+O Monero resolve o problema dos olhares curiosos, principamente graças a sua [fungibilidade](https://ww.getmonero.org/resources/moneropedia/fungibility.html). O atributo da fungibilidade torna suas transações anônimas para que você possa exercer suas liberdades sem ninguém te vigiar. Sejam esses: instituições financeiras, governos ou seu local de trabalho, o Monero assegura que você você mantenha seu dinheiro privado, como deve ser.
+
+Ao contrário do Monero, o Bitcoin é classificado como não-fungível, o que significa que o histórico de quaisquer moedas pode ser rastreado de volta até seu ponto de criação (quando foi minerada pela primeira vez). Qualquer endereço/carteira que fez transações pode ser rastreada no livro razão público. Caso uma moeda tenha sido usada em atividades ilegais no passado, isso torna-se parte do histórico dela.
+
+O Monero, em contrapartida, remove esse problema de um livro razão público e permanente. Um Monero (XMR) pode ser substituído por qualquer outro de mesmo valor. Isso significa que não existe um identificador único capaz de relacionar as transações de uma única moeda XMR ao longo do tempo. Adicionalmente, não existe o conceito nem o problema de se ter um XMR menos "puro" por causa do seu histórico manchado.
+
+Transações privadas e reserva de riqueza permanecerão sendo um importante direito para ambos indivíduos e empresas nas gerações futuras. Cada vez mais, temos visto governos que aproveitam-se dos poderes que lhe são dados, espionando seus cidadãos e de outras nações mesmo sem esses terem infrigindo quaisquer leis. Governos usam esses poderes anti-privacidade para neutralizar os, por eles denominados, grupos perigosos, ativistas e causas.
```

Comment:
Plural: "Transações privadas e **reservas** de riqueza"
"sendo um importante direito tanto para indivíduos quanto para empresas nas gerações futuras."
Remove the comma "Cada vez mais temos visto"
"governos que se aproveitam"
"espionando seus cidadãos ou até mesmo cidadãos de outras nações  ,(comma)  mesmo sem esses terem.."